foodalert/templates/base.html

```diff
@@ -6,6 +6,7 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <script>
             var netid = '{{ netid }}';
+            var logoutUrl = "{{ logout_url }}"
```

Comment:
Should we use double or single quotes for these variables? (just for the sake of consistency)